mw/index.d.ts

```diff
@@ -28,7 +29,61 @@ declare global {
          * global `window` object.
          * @see https://doc.wikimedia.org/mediawiki-core/master/js/#!/api/mw-property-config
          */
-        const config: mw.Map;
+        const config: mw.Map<{
+            debug: boolean;
+            skin: string;
+            stylepath: string;
+            wgArticlePath: string;
+            wgCaseSensitiveNamespaces: string[];
+            wgContentLanguage: string;
+            wgContentNamespaces: number[];
+            wgDBname: string;
+            wgEnableAPI: boolean;
+            wgEnableWriteAPI: boolean;
+            wgExtensionAssetsPath: string;
+            wgFormattedNamespaces: Record<string, string>;
+            wgNamespaceIds: Record<string, string>;
```

Comment:
```suggestion
            wgFormattedNamespaces: Record<number, string>;
            wgNamespaceIds: Record<string, number>;
```

Namespace IDs are numbers.

Reply:
oops, the documentation doesn't specify so the scraper script just turns `Object` into `Record<string, <string>`, should've checked for that